student-info-system/scheduler/scheduler.py

```diff
@@ -0,0 +1,21 @@
+from django.utils import timezone
+
+import pytz
+from apscheduler.schedulers.background import BackgroundScheduler
+
+from sis.models import Student, Message, Profile
+
+
+def academic_probation_check():
+    students = Student.objects.all()
+    profile = Profile.objects.get(user__username='zeus')
+    for student in students:
+        if student.gpa() < 2.0:
```

Comment:
this will say this every semester, even if it's "still" below 2.0 (the second time). Hope that's ok!

Reply:
yes usually they notify you until you fix it and sometimes several times per semester(not that I would know)